api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/dynamodb/AbstractDynamoDbProperties.java

```diff
@@ -45,6 +45,22 @@
      */
     private long writeCapacity = 10;
 
+    /**
+     * Billing mode specifies how you are charged for read and write throughput
+     * and how you manage capacity.
+     *
+     * Accepted values are {@code PROVISIONED, PAY_PER_REQUEST}.
+     *
+     * Defaults to {@code PROVISIONED}
```

Comment:
Please remove this line, and all references to the amazon docs. These refs are not maintainable in the long-term, and are not visible anywhere in the docs usually.